src/react/components/common/keyboardBinding/keyboardBinding.test.tsx

```diff
@@ -13,7 +13,7 @@ describe("Keyboard Binding Component", () => {
 
     const defaultProps: IKeyboardBindingProps = {
         accelerator: "Ctrl+1",
-        onKeyDown: onKeyDownHandler,
+        onKeyEvent: onKeyDownHandler,
```

Comment:
maybe this should also be rename `onKeyEventHandler`

Reply:
I think I'll keep it without the `Handler` just to keep it consistent with other event handlers like `onClick`, `onCancel`, etc.